cves/CVE-2013-6627.yml

```
@@ -4,21 +4,23 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE: 
+CWE: |
```

the CWE value should just be the integer ID.
```suggestion
CWE: 119
```